chat-components/src/services/BroadcastService.ts

```diff
@@ -2,16 +2,40 @@ import { Subject } from "rxjs";
 import { ICustomEvent } from "../interfaces/ICustomEvent";
 import { filter } from "rxjs/operators";
 import { BroadcastChannel } from "broadcast-channel";
-import { BROADCAST_CHANNEL_NAME } from "../common/Constants";
```

Comment:
If not used anymore, can we remove from Constants?

Reply:
removed this